code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -37,6 +37,10 @@
 	var/banana_type = /obj/item/grown/bananapeel
 	var/attack_reagent
 
+/mob/living/simple_animal/hostile/retaliate/clown/Initialize(mapload)
+	. = ..()
+	faction |= "clown"
```

Comment:
and im pretty sure here you can just do `faction = list("clown") ` in the type definition